Assets/Scripts/Singleton.cs

```diff
@@ -43,7 +43,7 @@ public static T Instance
                         singletonObject.name = $"{typeof(T).ToString()} (Singleton)";
 
                         // Make instance persistent.
-                        DontDestroyOnLoad(singletonObject);
+                        // DontDestroyOnLoad(singletonObject);
```

Comment:
Just curious what this does!

Reply:
`DontDestoryOnLoad` makes it so the object doesn't get destroyed when you change scenes, but I saw that may have caused errors when switching from Start Menu to Main scene back-and-forth so I'm just turning it off for now until I can look into it more.